docs/guides/react-components/_install-plain-java.md

```diff
@@ -6,7 +6,7 @@
 For this installation, you must have:
 
 - OpenJDK 21+ locally installed
-- Camunda `8.7.0-alpha3` or later
+- Camunda `8.6.0-alpha2` or later
```

Comment:
@hisImminence these are technically the preview for 8.8, so can we adjust to `8.7.0-alpha2`?